### DISCUSSION TITLE
Portuguese Novel

### DISCUSSION THREAD

**Discussion by Chronnos2005QM** [lnreader/lnreader#966] [Ideas]
Please, They provide plugins or extensions in Portuguese. There are no novels in Portuguese on the app 🙏🏾

**Comment by rajarsheechatterjee** [comment]
Create a Source Request in [LNReader/lnreader-sources](https://github.com/LNReader/lnreader-sources/issues/new?assignees=&labels=Source+Request&projects=&template=request_source.yml) repository.
